draft-ietf-httpbis-digest-headers.md

```diff
@@ -892,7 +910,6 @@ Content-Type: application/json
 Accept: application/json
 Accept-Encoding: identity
 Digest: sha-256=bWopGGNiZtbVgHsG+I4knzfEJpmmmQHf7RHDXA3o1hQ=
-Location: /books/123
```

Comment:
Is this intended? If so, the prose above also needs to be changed to remove reference to Location.

Reply:
@LPardue this was a typo because `Location` is a response header, while this is the request. Is that ok?

Reply:
Yep, all good